CHANGELOG.asciidoc

```diff
@@ -343,6 +343,12 @@ https://github.com/elastic/beats/compare/v5.2.2...v5.3.0[View commits]
 - The `symlinks` and `harverster_limit` settings are now GA, instead of experimental. {pull}3525[3525]
 - close_timeout is also applied when the output is blocking. {pull}3511[3511]
 - Improve handling of different path variants on Windows. {pull}3781[3781]
+- Restructure input.Event to be inline with outputs.Data {pull}3823[3823]
```

Comment:
Sorry to bug you again, but this seems to have been messed up during rebasing. Unfortunately happens from time to time :-(